test/e2e/node/mount_propagation.go

```diff
@@ -85,6 +85,9 @@ var _ = SIGDescribe("Mount propagation", func() {
 		// tmpfs to a subdirectory there. We check that these mounts are
 		// propagated to the right places.
 
+		hostExec := utils.NewHostExec(f)
```

Comment:
Just an observation for generalizing this - I think the hostexec pod should set `hostNetwork`, `hostIPC`, and `hostPID` to true

Reply:
[I think we can leave generalizing for a follow up] ?